libs/sr/ui/src/Character/LocationAutocomplete.tsx

```diff
@@ -1,17 +1,17 @@
 import type { GeneralAutocompleteOption } from '@genshin-optimizer/common/ui'
 import { GeneralAutocomplete } from '@genshin-optimizer/common/ui'
-import type { CharacterLocationKey } from '@genshin-optimizer/sr/consts'
-import { allCharacterLocationKeys } from '@genshin-optimizer/sr/consts'
+import type { CharacterKey } from '@genshin-optimizer/sr/consts'
+import { allCharacterKeys } from '@genshin-optimizer/sr/consts'
 import { Skeleton } from '@mui/material'
 import { Suspense, useCallback, useMemo } from 'react'
 import { useTranslation } from 'react-i18next'
 import { useDatabaseContext } from '../Context'
 
 type LocationAutocompleteProps = {
-  locKey: CharacterLocationKey | ''
-  setLocKey: (v: CharacterLocationKey | '') => void
+  locKey: CharacterKey | ''
```

Comment:
does not have to be this PR, but I would prefer `CharacterKey | null` here as well.

Reply:
https://github.com/users/frzyc/projects/2?pane=issue&itemId=76062503